components/file_preview_modal/file_preview_modal_main_actions/file_preview_modal_main_actions.test.tsx

```diff
@@ -98,4 +99,20 @@ describe('components/file_preview_modal/file_preview_modal_main_actions/FilePrev
         overlayWrapper.simulate('click');
         expect(spy).toHaveBeenCalledTimes(1);
     });
+
+    test('should not get public api when public links is disabled', async () => {
+        const spy = jest.spyOn(fileActions, 'getFilePublicLink');
```

Comment:
No, not when using useDispatch. That's why I prefer using connect/mapStateToProps instead of useDispatch/useSelector myself since then we can just pass in a `jest.fn` as the action instead of having to use a spy.